stdlib/public/core/Sort.swift

```diff
@@ -400,13 +400,13 @@ internal func _merge<Element>(
     // After moving elements, the storage and buffer look like this, where
     // `x` is uninitialized memory:
     //
-    // Storage: [4, 4, 7, 8, 9, 6, x, x,  x,  x,  x]
-    //                          ^  ^                 ^
-    //                    srcHigh  destLow        destHigh (past the end)
+    // Storage: [4, 4, 7, 8, 9, 16, x, x,  x,  x,  x]
+    //                           ^  ^                 ^
+    //                     srcHigh  destLow        destHigh (past the end)
```

Comment:
Yeah, the original is odd for a couple reasons:

1. That `6` isn't in the right spot; we wouldn't be merging a range of elements that isn't ordered correctly. (You fixed that by changing it to `16`)
2. `srcHigh` and `destLow` point to the same position, not consecutive ones. Looking at the loop below, they're actually _always_ the same (because we move elements from the top of the initialized "Storage" region, which shifts the uninitialized region downward). We could eliminate one if that would make things clearer, but they shouldn't depict two different pointers in this diagram.

Reply:
Yeah I wasn’t sure if there was a reason for doing it that way.

We could probably do something with, like, slashes as angled lines to show they’re both pointing to the same place.

Reply:
Okay I went with a slash, not as an angled line, but as an actual slash.